packages/components/src/slot-fill/bubbles-virtually/fill.js

```diff
@@ -1,16 +1,21 @@
 /**
  * WordPress dependencies
  */
-import { useRef, useEffect, createPortal } from '@wordpress/element';
+import { useRef, useState, useEffect, createPortal } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import useSlot from './use-slot';
 
+function useForceUpdate() {
+	const [ , setState ] = useState( {} );
+	return () => setState( {} );
+}
+
 export default function Fill( { name, children } ) {
 	const slot = useSlot( name );
-	const ref = useRef();
+	const ref = useRef( useForceUpdate() );
```

Comment:
Would it be a bit more clear if this was named 
```suggestion
	const ref = useRef( { rerender: useForceUpdate() } );
```

or force update or something like that